keyboards/keycapsss/kimiko/rules.mk

```diff
@@ -1,15 +1,5 @@
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = no            # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
-SPLIT_KEYBOARD = yes
 
-DEFAULT_FOLDER = keycapsss/kimiko/rev1
```

Comment:
Default folder needs adding back, or the file removing

Reply:
Ok, but can we set rev2 as default?

Reply:
Not without moving this PR to develop.

Reply:
Ok then it's fine and i can change it maybe later.

Reply:
Fauxy (@fauxpark) makes a fair point here regarding which branch to target for this PR.

To keep to PR Contributing guidelines - could this PR be exclusively for adding `rev2` board to `master` branch, and the rejigging of `rev1` moved to a separate PR targeting `develop` branch?

Reply:
@BenRoe 
I'd ask collaborater(s) to confirm the following: my understanding is changes in develop branch are merged to master branch every 3 month cycle (next cycle is last week in August)

Reply:
In my opinion (i.e. not speaking for QMK)- It would be easier to rebase/target this PR to develop branch, but if there is a commercial tie/dependency to this PR then i would recommend splitting it as discussed.

Reply:
If it get's merged last week in august, it's fine.
Sadly i do not have the Git skills to split this PR.

Reply:
> If it get's merged last week in august, it's fine. Sadly i do not have the Git skills to split this PR.

Best place to start would be revert changes to rev1 directory. By revert, take a copy of rev1 directory from master branch and save it somewhere other than qmk_firmware directory.

You could run `git checkout origin/master` to get a local copy of master branch, copy said directory. Then switch back to your branch locally using `git checkout kimiko_rev2_Ex3c`.

Replace rev1 directory with copied rev1 directory, then git push.

For the separate rev1 changes branch
Open your forked qmk_firmware repo on GitHub.  
View all branches from branch drop down.  
Click new branch button.  
Give this new branch a name and source from your forked develop branch if it's up to date, else source from qmk's develop branch. 

You'll need to then run `git pull` on your local machine to make your local machine aware of this new branch.
Run `git checkout -b` [newBranchName] `origin/`[newBranchName]  
Replace [newBranchName] with the name of the new branch created in GitHub.

You know the drill for commit and pushing so all good. When you're making pull request for this new branch to QMK, be sure to target qmk's develop branch 😉

Reply:
this suggestion still stands to resolve linting issues.
Further more, contents of this file should just contain DEFAULT_FOLDER (i.e. remove everything else)

Reply:
Default folder still needs to be specified